service.go

```diff
@@ -61,7 +69,7 @@ func (s *Service) Run(ctx context.Context) error {
 
 func writeErrorReply(w io.Writer, err error) error {
 	responseNotOk := birch.EC.Int32("ok", 0)
-	errorDoc := birch.EC.String("error", err.Error())
+	errorDoc := birch.EC.String("errmsg", err.Error())
```

Comment:
should we errcheck this?

Reply:
This doesn't expect a nil error, but I added a nil check anyways.